test/controllers/DownloadMetadataControllerSpec.scala

```diff
@@ -106,6 +106,30 @@ class DownloadMetadataControllerSpec extends FrontEndTestHelper {
       csvList.last("File Name") must equal("FileName2")
     }
 
+    "download the csv for rows with multiple values" in {
+      val customProperties = List(
+        customMetadata("TestProperty1", "Test Property 1"),
+        customMetadata("TestProperty2", "Test Property 2"),
+        customMetadata("FileName", "File Name")
+      )
+      val metadataFileOne = List(
+        FileMetadata("TestProperty1", "TestValue1File1"),
+        FileMetadata("TestProperty2", "TestValue2File1"),
+        FileMetadata("TestProperty2", "TestValue2File2"),
```

Comment:
Maybe ren-name the second value slightly to: `TestValue3File1` just to make clear this property is also for file one.